CMake Error

### Priority

High

### Area

- [ ] Datapack
- [X] Source
- [ ] Map
- [X] Other

### What happened?

Hello friends.

Config: Ubuntu 22.02

I'm starting the canary project and in the 5 steps Configure and build
When i paste cmake -DCMAKE_TOOLCHAIN_FILE=~/vcpkg/scripts/buildsystems/vcpkg.cmake .. --preset linux-release

Show this error:
![image](https://github.com/opentibiabr/canary/assets/48865229/666ab3e0-7510-4eaf-a828-651503f907dd)

But, searching in google i find one solutions what is change directory 
-DCMAKE_TOOLCHAIN_FILE=~/vcpkg/scripts/buildsystems/vcpkg.cmake
to:
-DCMAKE_TOOLCHAIN_FILE=/home/user/vcpkg/scripts/buildsystems/vcpkg.cmake

but it didn't solve
![image](https://github.com/opentibiabr/canary/assets/48865229/61d02afc-42ef-4e5d-b023-c3587ae7dc08)

Does anyone know what can it be?

Thanks,
Gabriel (M1113R)


### What OS are you seeing the problem on?

Linux

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

Tive o mesmo problema. Ubuntu 22.04
Vcpkg atualizado, tentei até 3x apagando tudo e tentando novamente.

![image](https://github.com/opentibiabr/canary/assets/89327037/7694c7f3-570b-45bb-9f94-544ef1e04226)

[install-x64-linux-dbg-out.log](https://github.com/opentibiabr/canary/files/14216791/install-x64-linux-dbg-out.log)
[vcpkg-manifest-install.log](https://github.com/opentibiabr/canary/files/14216793/vcpkg-manifest-install.log)

DBG out linha 116:
/home/paiva/vcpkg/buildtrees/opentelemetry-cpp/src/v1.12.0-c900296c5a.clean/api/include/opentelemetry/version.h:25:4: fatal error: error writing to /tmp/cc2SpQrj.s: No space left on device

Na verdade, existe espaço, mas alguma configuração parece estar errada no vcpkg ou no ubuntu. Não tenho tanto conhecimento em ubuntu pra identificar o problema.

--- a/src/server/network/protocol/protocolgame.cpp
+++ b/src/server/network/protocol/protocolgame.cpp
@@ -4133,15 +4133,21 @@ void ProtocolGame::sendTextMessage(const TextMessage &message) {
 		}
 		case MESSAGE_HEALED:
 		case MESSAGE_HEALED_OTHERS: {
-			msg.addPosition(message.position);
-			msg.add<uint32_t>(message.primary.value);
-			msg.addByte(message.primary.color);
+			if (!oldProtocol) {
+				msg.addPosition(message.position);
+				msg.add<uint32_t>(message.primary.value);
+				msg.addByte(message.primary.color);
+			}
 			break;
 		}
 		case MESSAGE_EXPERIENCE:
 		case MESSAGE_EXPERIENCE_OTHERS: {
 			msg.addPosition(message.position);
-			msg.add<uint64_t>(message.primary.value);
+			if (!oldProtocol) {
+				msg.add<uint64_t>(message.primary.value);
+			} else {
+				msg.add<uint32_t>(message.primary.value);
+			}
 			msg.addByte(message.primary.color);
 			break;
 		}

--- a/vcpkg.json
+++ b/vcpkg.json
@@ -16,7 +16,7 @@
     {
       "name": "opentelemetry-cpp",
       "default-features": true,
-      "features": ["otlp-http", "prometheus"]
+      "features": [ "otlp-http", "prometheus" ]
     },
     "parallel-hashmap",
     "protobuf",
@@ -25,7 +25,7 @@
     "zlib",
     {
       "name": "libmariadb",
-      "features": ["mariadbclient"]
+      "features": [ "mariadbclient" ]
     },
     {
       "name": "gmp",
@@ -36,5 +36,5 @@
       "platform": "windows"
     }
   ],
-  "builtin-baseline": "98a562a04cd03728f399e79e1b37bcccb5a69b37"
+  "builtin-baseline": "095ee06e7f60dceef7d713e3f8b1c2eb10d650d7"
 }

